server/lib/utils.js

```diff
@@ -513,6 +513,12 @@ export const getBearerTokenFromRequestHeaders = req => {
   }
 };
 
+export const getBearerTokenFromCookie = req => {
+  return req?.cookies?.accessTokenPayload && req?.cookies?.accessTokenSignature
+    ? [req.cookies.accessTokenPayload, req.cookies.accessTokenSignature].join('.')
+    : null;
+};
+
```

Comment:
We already have that: `getTokenFromCookie` in `lib/auth.ts`

Reply:
Thats on the frontend codebase.

Reply:
My bad, indeed! The file that confused me here (because we have one with a similar name in frontend & API), `server/lib/auth.js` is probably a better fit than  `server/lib/utils.js`.